### ISSUE TITLE
Multiple configuration files created

### ISSUE THREAD

**Issue by rohankrishnadev** [robertdebock/ansible-role-vault#1]
## Describe the bug
The role this is dependent on (`robertdebock.hashicorp`) seems to create a config file under `/etc/vault.d/vault.hcl` where as this role creates and uses the config file `/etc/vault/config.hcl`. It might be cleaner to have one config file or at least have them under the same directory.

I would suggest taking a backup of `/etc/vault.d/vault.hcl` if it exists and overwriting it. I can send a pull request if you'd like to do it that way.

## Playbook
My playbook consists of:

```yaml
---
- hosts: all

  vars_files:
    - vars/main.yml

  roles:
    - robertdebock.bootstrap
    - robertdebock.hashicorp
    - robertdebock.vault
```

My `vars/main.yml` consists of:

```yaml
---
bootstrap_user: vagrant

# robertdebock.hashicorp variables
hashicorp_products:
  - vault

# robertdebock.vault variables
vault_storages:
  - name: raft
    path: /opt/vault/data
    node_id: node1
```

## Output

Show at least the error, possible related output, maybe just all the output.

## Environment

- Control node OS: MacOS Catalina 10.15.7
- Control node Ansible version: ansible 2.9.11
- Managed node OS: Ubuntu 20.04.1 LTS

Please consider [sponsoring me](https://github.com/sponsors/robertdebock).

**Comment by robertdebock** [comment]
Fully agreed. I'll work on getting the configuration in one directory.

Thanks again @rohankrishnadev!

**Comment by robertdebock** [comment]
I'll let CI test the changes and release a new version later, likely tomorrow.

Thanks for the heads up, certainly an improvement!

**Comment by robertdebock** [comment]
Just released version 3.0.0
